fibonacci.py

```diff
@@ -0,0 +1,22 @@
+# Automation onboarding
+# Fibonacci series: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144…
+
+def fibonacci(maxN):
```

Comment:
For some reason this code is new in this PR, I think it should be just in the other PR, to separate responsibilities, fixes, new features, new tests separated

Reply:
I found out why! I accidentally grab all the files when creating this new branch. Is there a way to delete unwanted files from this PR?